### DISCUSSION TITLE
Help with API Page not working

### DISCUSSION THREAD

**Discussion by cyaninja** [supabase/supabase#5639] [Questions] [answered]
Hi I am trying to self host supabase (via docker) but the API page does not work at all. I get the error in this screenshot

<img width="453" alt="image" src="https://user-images.githubusercontent.com/38188189/155386130-cf792742-fff7-402c-bf8b-ca803eb92872.png">

On Inspect Console, I see this:
Failed to load resource: net::ERR_NAME_NOT_RESOLVED
<img width="751" alt="image" src="https://user-images.githubusercontent.com/38188189/155386034-da45c4d6-407a-4099-9777-2e3c040a9dc7.png">

I already saw the Issue raised here [5402](https://github.com/supabase/supabase/issues/5402) but I am unable to replicate the solution. I tried setting both the .env variable to use host.docker.internal as well as added the extra_hosts to the docker compose yml file as suggested in that thread

Once I did that, I get this error instead:

<img width="745" alt="image" src="https://user-images.githubusercontent.com/38188189/155388169-a071bad7-ca19-46cc-817e-20b7ccf327d2.png">

I must be doing something wrong! But don't know what it is. I am trying to do this on Ubuntu 20.04

**Comment by cjmadushka** [comment, marked as answer]
This Worked For Me 
![image](https://user-images.githubusercontent.com/79196915/155449290-5a5c73ad-48de-48b2-ba8c-a07cf91533c5.png)

**Reply by cyaninja** [reply]
A million thanks!
